x-pack/plugins/ml/common/types/annotations.ts

```diff
@@ -73,8 +73,15 @@ export interface Annotation {
   annotation: string;
   job_id: string;
   type: ANNOTATION_TYPE.ANNOTATION | ANNOTATION_TYPE.COMMENT;
+  event?: string;
```

Comment:
Just clarified with Ben that the event will be omitted from the payload if it is `null` on the backend side. So we are guaranteed that if has a valid value, it will be shown, and if not, it will be omitted/undefined. [We based it on here.](https://github.com/elastic/elasticsearch/blob/62fa4cb319637caa69417335f9addfde7893f7b0/x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/annotations/Annotation.java#L340)